### PR TITLE
Update screenshots for new Cloud/Editor UI [HC-1496]

### PR DESCRIPTION
**Pull request by gorillagripcore**

### CODE REVIEW

File: content/Arduino Cloud/Cloud Editor/If-your-board-is-not-detected-by-Arduino-Cloud-Editor.md [hunk 1]
```diff
@@ -32,9 +32,7 @@ Learn how to troubleshoot possible issues when connecting a board to the Cloud E
 
 ### Classic Nano boards and boards with generic USB chips
 
-Classic Nano boards and some clone boards with generic USB chips will show up as **"Board"** in the board selector. When selecting a "Board" port you will be prompted to select the board type manually.
-
-![Cloud Editor with port name showing in the board selector ](img/WebEditor_port_no_board_name.png)
+Classic Nano boards and some clone boards with generic USB chips will show up as **"Board"** in the board selector. When selecting a **"Board"** port you will be prompted to select the board type manually.
```
Comment (seaxwi) [inline content/Arduino Cloud/Cloud Editor/If-your-board-is-not-detected-by-Arduino-Cloud-Editor.md:35]: We should test this with the new Cloud Editor.

File: content/Arduino Cloud/Cloud Editor/Import-your-sketchbook-and-libraries-to-the-Cloud-Editor.md [hunk 2]
```diff
@@ -3,48 +3,46 @@ title: "Import your sketchbook and libraries to the Cloud Editor"
...
+   - Linux: `$HOME/Arduino`.
 
-2. Zip your sketchbook:
+1. Zip your sketchbook
 
-   Make a `.zip` pack of your sketchbook, you should obtain a file called `Arduino.zip`. Make sure it is `.zip` format, any other archive formats will not work.
+   Make a `.zip` pack of your sketchbook, you should obtain a file called `Arduino.zip`. Make sure it is in `.zip` format, any other archive formats will not work.
 
-   ![Create ZIP file](img/ImportFiles1.jpg)
+   ![Create ZIP file](img/ImportFiles1.jpg)
```
Comment (seaxwi) [inline content/Arduino Cloud/Cloud Editor/Import-your-sketchbook-and-libraries-to-the-Cloud-Editor.md:30]: On another note, I would just remove the image:

* It's inconsistent with other images in the article (bad compression, different drop-shadow)
* It's instructive only for macOS users

**Review by seaxwi — CHANGES_REQUESTED** [review]
New images look great, some comments.

File: content/Arduino Cloud/Cloud Editor/If-your-board-is-not-detected-by-Arduino-Cloud-Editor.md (hunk 1 above)
Comment (gorillagripcore) [reply content/Arduino Cloud/Cloud Editor/If-your-board-is-not-detected-by-Arduino-Cloud-Editor.md:35]: image

**Review by seaxwi — APPROVED** [review]

**Pull request merged**